controllers/database_controller.py

```diff
@@ -54,6 +56,34 @@ def reset_ras_database():
     trans.commit()
 
 
+def reset_secure_message_database():
+    logger.info("Clearing down secure message database")
+    engine = create_engine(Config.SECURE_MESSAGE_DATABASE_URI)
+    connection = engine.connect()
+    trans = connection.begin()
+
+    with open('resources/database/database_reset_secure_message.sql', 'r') as sqlScriptFile:
+        sql = sqlScriptFile.read().replace('\n', '')
+
+    connection.execute(sql)
+    trans.commit()
+    connection.close()
+
+
+def create_secure_messages():
+    logger.info("Putting party database into known state")
```

Comment:
Is there a way we can do this via an api?

Reply:
I agree you should be hitting the SM API for this rather than the database directly I think.

Reply:
Also I guess the full UI is there to be able to do this so you could drive it off the UI unless there is a reason not to